app/models/provider.rb

```diff
@@ -65,8 +65,9 @@ def all_associated_accredited_providers_onboarded?
     accredited_providers.all?(&:onboarded?)
   end
 
-  def no_admin_users?
-    !(provider_permissions.exists?(manage_users: true) &&
+  def lacks_admin_users?
+    !(sync_courses &&
```

Comment:
I think this should be `sync_courses && !(...` — at the moment the `sync_courses` condition is backwards which means everything gets the "No admin user" label

<img width="869" alt="Screenshot 2021-04-15 at 17 03 32" src="https://user-images.githubusercontent.com/642279/114901286-8e5a2280-9e0c-11eb-83c2-72c04b51a1f6.png">